tests/test_client.py

```diff
@@ -451,3 +450,63 @@ def test_complete_client_visit(client: TestClient):
     response = client.post("/api/client/complete_client_visit", json=data)
     assert response
     assert response.ok
+
+    # 1. create Reception
+    doctor = Doctor.query.first()
+    # 2. add 3 visits
+    date = datetime.date.today()
+    time = datetime.datetime.utcnow().strftime("%m/%d/%Y, %H:%M:%S")
+    count = 1
+    for clientDB in Client.query.limit(3).all():
+        Visit(
+            date=date + datetime.timedelta(days=count),
+            start_time=datetime.datetime.strptime(time, "%m/%d/%Y, %H:%M:%S")
+            + datetime.timedelta(days=count),
+            end_time=datetime.datetime.strptime(time, "%m/%d/%Y, %H:%M:%S")
+            + datetime.timedelta(days=count + 2),
+            client_id=clientDB.id,
+            doctor_id=doctor.id,
+        ).save()
+        count = count + 1
+
+    visit: Visit = Visit.query.all()
+    assert visit
+    # assert len(visit) == 3
+
+    visit1: Visit = Visit.query.first()
+    assert visit1
+    visit3: Visit = Visit.query.get(3)
+    assert visit3
+
+    # 3. get visits for report
+    data = {
+        "type": "visit",
+        "start_time": visit1.start_time.strftime("%m/%d/%Y, %H:%M:%S"),
+        "end_time": visit3.end_time.strftime("%m/%d/%Y, %H:%M:%S"),
+    }
+    # 4. put date for filter report visits
+    response = client.post("/api/client/report_visit", json=data)
+    assert response
+    assert response.ok
+    # assert response.text
```

Comment:
really need comment or we can remove it?)

Reply:
Thank you for all your comments, I will fix everything :)